### ISSUE TITLE
Keras binary_crossentropy vs sklearn log_loss

### ISSUE THREAD

**Issue by thundo** [keras-team/keras#9219]
Hello, I'm using Keras 2.1.3 with tensorflow 1.5.0 and sklearn 0.19.1. 

I'm using a KerasClassifier with sklearn cross_validate/GridSearchCV for a multilabel problem. I use binary_crossentropy in my model and I'm trying to score sklearn cross_validation/parameter search with log_loss/neg_log_loss instead of accuracy. However I find quite a big difference between the two returned values.

```
from sklearn.metrics import log_loss
from keras import backend as K
from keras.losses import binary_crossentropy
print('Log_loss: {}'.format(log_loss(Y, Y_pred)))
print('Log_loss: {}'.format(
    K.eval(
        K.mean(
            binary_crossentropy(
                K.variable(Y),
                K.variable(Y_pred))
            )
        )
    )
)
```

Sklearn returns a log_loss of about 0.26, while Keras of about 0.03. Shouldn't these calls return the same value? I'm using the wrong score function for a multilabel problem?

For more background I first asked on [SO](https://stackoverflow.com/questions/48470539/keras-parameter-search-and-log-loss), but with no conclusive help till now.

Thank you for your time
